### ISSUE TITLE
Complete deck sheet

### ISSUE THREAD

**Issue by zeel01** [zeel01/monarch#11]
Complete the deck sheet design.

**Comment by Tommycore** [comment]
Having a sheet that displays all cards in a deck is going to come in *very* handy for all cases where cards have a text printed on them (i.e. "Play to make +2 damage"). The sheet could use some stronger visual cue though as to which cards have been drawn. If you take a moment you can make out which card was drawn from those eight in the picture below, and which wasn't, but it isn't immediately clear at a quick glance.

![image](https://user-images.githubusercontent.com/19580006/151180859-40bce11a-df5b-44b3-bcf6-cbf53917b255.png)

Another idea: How about having a view where the cards themselves are smaller, as to get more on the screen at the same time, but show a detailed view of the card on mouse hover?

Also - Is there a roadmap with how the deck sheet design is planned to look in the end?

**Comment by zeel01** [comment]
> Is there a roadmap with how the deck sheet design is planned to look in the end?

No, not really. For the deck/pile I'm really unsure the best approach. And for any design, I basically go strait from in my head to CSS - I'm not much of an artist, so I would spend more time on a mockup than I would just implementing it and seeing how it looks.
